README.md

```diff
@@ -17,26 +17,39 @@ This game uses `python 3.1.0`
 - Type `python tic_tac_toe.py` into your command line prompt while in this repo's directory.
 - The game will go through a series of prompts for a two-player TTT game.
 - You will be notified if you input an incorrect value and be re-prompted to correct the value.
-- The game will declare a winner and gives the option to play again.
+- The game will declare a winner and give the option to play again.
 
-### Testing:
+## Testing:
 
-This project uses [unittest](https://docs.python.org/3/library/unittest.html), a Python testing framework.
+This project uses [unittest][1], a Python testing framework.
```

Comment:
has everything been migrated over to `unittest` or are there still `pytest`s as well?  I may have missed it.

Reply:
I actually never ended up using `pytest`.  I thought I would but we found our solution with `unittest` instead.  If I end up using it in the future, I can add it back into the README but for now, only `unittest` is being used.

Reply:
ok, thanks.

Reply:
Did you see I used your README trick for links?!

Reply:
I did!  footnotes in markdown are wonderful! 🎉